spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../../lib/takeaway.rb'
+require_relative '../../lib/menu.rb'
+require 'twilio-ruby'
+require 'dotenv'
+
+describe "FEATURE SPEC" do
+  let(:takeaway) { Takeaway.new(Menu.new, Calculator.new,
+  Twilio::REST::Client.new("01234", "56789")) }
+  let(:choice) { {beef:2, rolls:3} }
+  let(:default_menu) { {ribs: 3,
+        beef: 4,
+        rolls: 3,
+        chips: 2,
+        pies: 5}
+        }
+  let(:confirm) {"Thank you! Your order was placed "\
+                  "and will be delivered before "\
+                 "#{(Time.now+ 60*60).strftime("%H:%M")}"
+                }
```

Comment:
Lint/BlockAlignment: `}` at 19, 16 is not aligned with `let(:confirm) {"Thank you! Your order was placed "\` at 16, 2.